src/Application_TEST.cc

```diff
@@ -84,16 +84,17 @@ TEST(ApplicationTest, IGN_UTILS_TEST_DISABLED_ON_WIN32(LoadPlugin))
   // Plugin path added programmatically
   {
     Application app(g_argc, g_argv);
+    app.AddPluginPath(std::string(PROJECT_BINARY_PATH));
```

Comment:
Is this necessary because Bazel isn't installing the plugins to `lib`? I have no idea of what's involved in this, but wouldn't it be preferable if the Bazel setup would install libraries there?